### ISSUE TITLE
Data trends (averages, graphs, etc.)

### ISSUE THREAD

**Issue by LTryon** [ashokkumar9892/patient-app-Backup#86]
Ester has additional views for you to see prior to building this out.  Please schedule meeting with her.  I do not need to be present.

**Comment by ester245** [comment]
Yes please. Ashok, please let me know when we can schedule a time to meet so that I can show you the functionality of the iGlucose website and everything we would like it to be like. I would like to provide access to patient accounts so that you can feel out the way the website is working.

**Comment by ashokkumar9892** [comment]
Sure we can do today around 1:30 pm, i believe you gave me like this one??
![image](https://user-images.githubusercontent.com/8000169/137506178-a5c5d9b5-f6da-4a34-b90a-2f830b3d61e4.png)

**Comment by ester245** [comment]
Yes we can meet at 1:30 today I might be a few minutes late because will be on a call with patient at 1:00pm. But I will let you know if I am late for any reason.

**Comment by ashokkumar9892** [comment]
np, i will wait in call.

Thanks

**Comment by ashokkumar9892** [comment]
join same meeting Feedback
Friday, October 15 · 8:30 – 9:00am
Google Meet joining info
Video call link: https://meet.google.com/uiq-cvux-zup

**Comment by ashokkumar9892** [comment]
bertharidley@apatternplus.com
https://connect.iglucose.com/login.php
Anonymous#123



This BG

**Comment by ashokkumar9892** [comment]
jcrane710@gmail.com
Anonymous123

this is for BP

**Comment by ester245** [comment]
I checked the chart for BG and it looks great, however, the graphing dates are sporadic and not helpful with that glitch. Can we fix this? see picture example. Can we make data show in charts in chronological order?

![image](https://user-images.githubusercontent.com/92586010/139925820-49c4b8b0-b054-4a10-aa6b-0d41c6b92ed7.png)

can you see the picture?

**Comment by ashokkumar9892** [comment]
Hi Ester,
Sorry Ester, i did not do anything much today because i am staying in
quarantine zone, but i ll take a look.
Sure this is a bug i ll fix this bug and let you know, This should be date
wise not date and time.

Thanks
Ashok-

**Comment by LTryon** [comment]
Hi Ashok,

How are you doing?  I hope you are feeling better soon. If your symptoms 
are mild, and since you work from home, will you be doing any work on 
this project while quarantined?  If not, when can we expect that you'll 
be back?

Actually, the graph needs both date and time, in the order that the 
readings occurred. Some patients will have more than one reading in a 
day.

Thanks!


---
Dr. Lilly Tryon, DNP, APN, FNP-BC, DipACLM, NBC-HWC, ACSM EP-C
Family Nurse Practitioner
Certified Lifestyle Medicine Professional

A Pattern Medical Clinic
5545 Little Debbie Pkwy
Ooltewah, TN 37363
423-455-2711
www.APatternClinic.com

"I have come that they may have life, and have it to the full."  John 
10:10, NIV

**Comment by ashokkumar9892** [comment]
Hi Dr Tryon,
Yes i am working from home but right now taking rest because my body does
not allow to do anything.

Will you be available tomorrow around 8:30 or friday we can have quick
meeting.


I got feedback from Ester and i am fixing chat as Iglucose like per date. I
will connect with you tomorrow to clarify more.

Thanks
Ashok-

**Comment by LTryon** [comment]
Friday morning is fine.  I do not work at the clinic on Thursdays.


---
Dr. Lilly Tryon, DNP, APN, FNP-BC, DipACLM, NBC-HWC, ACSM EP-C
Family Nurse Practitioner
Certified Lifestyle Medicine Professional

A Pattern Medical Clinic
5545 Little Debbie Pkwy
Ooltewah, TN 37363
423-455-2711
www.APatternClinic.com

"I have come that they may have life, and have it to the full."  John 
10:10, NIV
